docs/modules/ROOT/pages/references/release_notes.adoc

```diff
@@ -2,6 +2,59 @@
 
 TIP: This page lists notable changes in OpenShift releases which we find important. Reading release notes for you as a service.
 
+== OpenShift 4.11
+
+OpenShift version 4.11 is available.
```

Comment:
Since when? Having dates in release notes is crucial to me... See older release notes on this page how it was done.